spec.html

```diff
@@ -21598,16 +21637,31 @@
           </emu-note>
         </emu-clause>
 
+        <emu-clause id="sec-resolutionfailure" aoid="ResolutionFailure">
+          <h1>Runtime Semantics: ResolutionFailure( _module_, _resolveMode_ )</h1>
+          <p>The ResolutionFailure abstract operation performs the following steps:</p>
+          <emu-alg>
+            1. Assert: _module_.[[Status]] is `"uninstantiated"` or `"instantiating"`.
```

Comment:
|module| here is the exporter rather than the importer, so I think it's fine for this to be in any state, except possibly "errored".